spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -85,7 +92,8 @@ public SparkRowLevelOperationsTestBase(String catalogName, String implementation
                 "cache-enabled", "false" // Spark will delete tables using v1, leaving the cache out of sync
             ),
             "avro",
-            false
+            false,
+            "range"
```

Comment:
Can you please import and use the named constants for the distribution modes? That way if anybody goes to update things, their IDE will be able to help guide them to the right location.

Reply:
I think this is useful. I have updated.